code/game/objects/items/stunbaton.dm

```diff
@@ -230,3 +230,40 @@
 /obj/item/melee/baton/cattleprod/baton_effect()
 	if(sparkler.activate())
 		..()
+
+/obj/item/melee/baton/cattleprod/borg
+	name = "eletric arm"
+	desc = "Zap."
+	icon = 'icons/mob/robot_items.dmi'
+	icon_state = "elecarm"
+	hitcost = 1000

```

Comment:
wee woo wee woo

Not only is it now a stun prod, but it also costs 1/15 of the roundstart borgs cell? Thats a bit dumb.

Reply:
I'd like to point out I encouraged someone else to rework it as a baton just a couple days ago. I dont disagree with the stun rework, but I do disagree with the high cost.

Reply:
That cost is way too high, do remember that borgs don't have cuffs and need to use their stuns to keep people from escaping from a robo factory on cooldown and the only borg that *does* have cuffs doesn't have a stun (flashes are a very poor stun since they burn out and don't work on anyone intelligent enough to get flash protection).